plugins/worldinterface/include/worldproxy.h

```diff
@@ -195,6 +195,7 @@ class WorldProxy:public GazeboYarpPlugins::WorldInterfaceServer
    * @param id string that identifies object in gazebo (returned after creation)
    * @param link_name name of a fully scoped link (e.g. MODEL1::link)
    * @return true if success, false otherwise
+   *    attach OBJECT_NAME::object_parent_link1::....::object_end_link ROBOT_NAME::robot_parent_link1::....::robot_end_link
```

Comment:
This line out of nowhere in the docs is a bit confusing. I would add a "Example of use from yarp rpc" before the line or something similar.

Reply:
See https://github.com/robotology/gazebo-yarp-plugins/pull/288/commits/e13b5afa2b96aa7887c4849e353ae137f6c530be